src/App.tsx

```diff
@@ -1,12 +1,24 @@
 import React from 'react';
+import { TodoList } from './components/TodoList/todo_list';
+import { Todo } from './components/Type/Todo';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+
+const preparedTodos: Todo[] = [...todos].map((ell) => {
```

Comment:
```suggestion
const preparedTodos: Todo[] = todos.map((todo) => {
```
No need to create the copy.
BTW, what is 'ell'? 🤔